stdlib/area/area.lua

```diff
@@ -314,4 +330,28 @@ function Area.tostring(area)
     return 'Area {'..left_top..', '..right_bottom ..'}'
 end
 
+--- Converts an entity and its collision_box to the area around it
+-- @see Entity.to_collision_area
+-- @tparam LuaEntity entity
+-- @treturn LuaBoundingBox
+function Area.to_collision_area(entity)
+    fail_if_missing(entity, "missing entity argument")
```

Comment:
I actually tried to make it a reference as per the example but both files require each other leading to infinite loop. 

I and others find ourselves 99% of the time using Area and only needing Entity for getting the collision area offset.